### PR TITLE
Checkup: Use container disk image instead of PVC

### PR DESCRIPTION
**Pull request by orelmisan**
Currently, the VMI spec is using a PVC as the storage of the VMI.
This was a temporary in order to start the development process.
Replace the PVC volume with the checkup's custom container disk image [1].

Add the `vmUnderTestContainerDiskImage` parameter to the user-supplied ConfigMap.

[1] quay.io/kiagnose/kubevirt-realtime-checkup-vm

### CODE REVIEW

**Comment by orelmisan** [comment]
Tested against an OpenShift Virtualization 4.15 cluster.

File: tests/checkup_test.go [hunk 1]
```diff
@@ -250,18 +250,24 @@ func newRoleBinding(name, serviceAccountName, roleName string) *rbacv1.RoleBindi
 }
 
 func newConfigMap() *corev1.ConfigMap {
+	testConfig := map[string]string{
+		"spec.timeout":                                 "15m",
+		"spec.param.targetNode":                        "",
+		"spec.param.guestImageSourcePVCNamespace":      testNamespace,
+		"spec.param.guestImageSourcePVCName":           "centos-8-rt",
```
Comment (RamLavi) [inline tests/checkup_test.go]: off-topic: is it assumed that the guestImageSourcePVCName PVC is pre-installed on the cluster?

Comment (orelmisan) [reply tests/checkup_test.go]: After this PR there is no need to pass these parameters.
[Removed](https://github.com/kiagnose/kubevirt-realtime-checkup/compare/ffb8320c5db4824da4ad21ed427508675af53f1f..85160e6a89b2d0a3ceb4fa04549688ea4ede5be5) them in the last commit.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kubevirt-realtime-checkup/compare/ffb8320c5db4824da4ad21ed427508675af53f1f..85160e6a89b2d0a3ceb4fa04549688ea4ede5be5): Remove leftover PVC params from e2e test.

**Review by RamLavi — APPROVED** [review]
/lgtm

**Pull request merged**